src/main.h

```diff
@@ -217,6 +217,9 @@ bool GetTransaction(const uint256& hash, CTransaction& tx, uint256& hashBlock, b
 bool GetOutput(const uint256& hash, unsigned int index, CValidationState& state, CTxOut& out);
 /** Find the best known block, and make it the tip of the block chain */
 
+//! for accessibility from consensus/zerocoin_verify.cpp
+CAmount GetInvalidUTXOValue();
```

Comment:
Instead of exporting it from main, wouldn't it make more sense to just implement it directly in consensus/zerocoin_verify.cpp? It's only accessed from `RecalculatePIVSupply`.

Reply:
moved it